cdap-ui/app/cdap/components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/LiveConfigurationGroupButton/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+
+import withStyles, { StyleRules } from '@material-ui/core/styles/withStyles';
+
+import IconButton from '@material-ui/core/IconButton';
+import ImportContactsIcon from '@material-ui/icons/ImportContacts';
+import { LiveViewMode } from 'components/PluginJSONCreator/Create/PluginJSONMenu';
+import Tooltip from '@material-ui/core/Tooltip';
+import classnames from 'classnames';
+
+const styles = (theme): StyleRules => {
+  return {
+    buttonTooltip: {
+      fontSize: '13px',
+      backgroundColor: theme.palette.grey[50],
+    },
+    liveConfigurationGroupButton: {
+      '&:focus': {
+        outline: 'none',
+      },
+    },
+    clickedButton: {
+      backgroundColor: 'rgb(26,115,232,0.2)',
```

Comment:
Can we get this from theme.palette? (It looks like theme.palette.primary.main)

Reply:
Resolved